src/commands/cloud-run/flare.ts

```diff
@@ -0,0 +1,254 @@
+import IService = google.cloud.run.v2.IService
+import fs from 'fs'
+import path from 'path'
+import process from 'process'
+import util from 'util'
+
+import {ServicesClient} from '@google-cloud/run'
+import {google} from '@google-cloud/run/build/protos/protos'
+import chalk from 'chalk'
+import {Command} from 'clipanion'
+import {GoogleAuth} from 'google-auth-library'
+
+import {API_KEY_ENV_VAR, CI_API_KEY_ENV_VAR, FLARE_OUTPUT_DIRECTORY, FLARE_ZIP_FILE_NAME} from '../../constants'
+import {createDirectories, deleteFolder, writeFile, zipContents} from '../../helpers/fileSystem'
+import {sendToDatadog} from '../../helpers/flareFunctions'
+import {requestConfirmation} from '../../helpers/prompt'
+import * as helpersRenderer from '../../helpers/renderer'
+
+import {maskEnvVar} from '../lambda/functions/commons'
+
+const SERVICE_CONFIG_FILE_NAME = 'service_config.json'
+
+export class CloudRunFlareCommand extends Command {
+  private isDryRun = false
+  private service?: string
+  private project?: string
+  private location?: string
```

Comment:
I am unsure whether this should be called `location` or `region`. It seems GCP uses both interchangeably.
For example, the `gcloud run` CLI tool calls them regions (example: `gcloud run regions list`, `gcloud run deploy ... --region us-east1`).
However, the service configuration YAML calls them `location` instead.

Maybe we support both?